src/main/resources/META-INF/plugin.xml

```diff
@@ -41,17 +41,17 @@
                  defaultValue="720000"
                  description="Snyk timeout (milliseconds) to wait for results during scan"/>
 
-    <registryKey key="snyk.code.report.false.positives.enabled"
+    <registryKey key="snyk.preview.snyk.code.ls.enabled"
```

Comment:
Question:  I'm guessing we can remove the `snyk.code.report.false.positives.enabled` feature flag? I think I saw it used in a function called `isReportFalsePositivesEnabled` but I might not understand how this works well enough

Reply:
Code path is not used

Reply:
This is addressed in https://github.com/snyk/snyk-intellij-plugin/pull/479